app/assets/javascripts/components/cloud_volume_backup/cloud-volume-backup-form.js

```diff
@@ -56,9 +56,6 @@ function cloudVolumeBackupFormController(miqService, $http) {
 
   function getVolumeFormDataComplete(response) {
     vm.volume_choices = response.data.volume_choices;
-    if (foundVolumes()) {
```

Comment:
You don't have to remove this feature.

You can just move it *after* the `vm.modelCopy = ...` line, so that the save button doesn't get greyed out

Reply:
So then I should change the `foundVolumes` function to return true if there is exactly one choice, instead of removing it. Right?

Reply:
I am not sure. The original behavior: the first item was always selected - because of this code I am trying to delete here; and Save button was always enabled. The BZ's expected behavior: it is not filled in in the description of the BZ, it is missing. But from another comments: Save button disabled when first time in and something like 'Choose a volume' present and chosen in the dropdown.
So if we want to keep this code here for some reason, it makes sense for me to change it so that the item will be selected by default in the drop down only if there will be only one item in the drop down, not more of them. So I would delete the code as it is here, or to change it to something like
```
...
   return vm.volume_choices.length === 1;
```
What do you think? I cannot see the difference between setting `vm.cloudVolumeBackupModel.volume` before or after `vm.modelCopy =`.

Reply:
Hmm, oh.. Thank you! I realized..  I probably got not the best db/reproducer from the reporter of that bug: there's no only one Volume in the system (see my screenshots) so I probably misunderstood the whole problem. Thanks!

Reply:
No problem :). The rule for this pre-select behaviour should be that there is no difference between pre-select doing it and the user doing it. 

(And if the user did it, it would not end up in modelCopy in this case, which is where the bug comes from IMO.)

Reply:
Yep, I get that. As soon I get proper data or reproducer, I will make changes, test them and update this PR. It looks like the bug is not reproducible if there are more Cloud Volumes.